package.json

```diff
@@ -149,6 +149,7 @@
     "text-encoding": "^0.7.0",
     "toastr": "^2.1.4",
     "triple-beam": "^1.3.0",
+    "ts-essentials": "^7.0.1",
```

Comment:
just for `UnreachableCaseError`?

Reply:
Us amongst the team has been contemplating using this package for a while for stuff such as `XOR` types which is not available in `type-fest`, so might as well start the ball rolling and use it for `UnreachableCaseError` first. But I see that you have a new PR #943 so I'll remove it for now :)